akka-typed/src/main/scala/akka/typed/ActorRef.scala

```diff
@@ -74,6 +68,15 @@ abstract class ActorRef[-T](_path: a.ActorPath) extends java.lang.Comparable[Act
 }
 
 object ActorRef {
+
+  implicit final class ActorRefScalaTell[-T](val ref: ActorRef[T]) extends AnyVal {
+    /**
+     * Send a message to the Actor referenced by this ActorRef using *at-most-once*
+     * messaging semantics.
+     */
+    def !(msg: T): Unit = ref.tell(msg)
+  }
```

Comment:
I'm not worried about performance (it should not add anything). I'm just thinking if the magic is unnecessary, but the small advantage of hiding it from java might be worth it.